### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by AtillaTahak**
-- added linter
-- Created Skeletor for HTML
-- Created Component 
-- Added some style
-- Added React Router
-- Added Navbar

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      <p>Suzanne Collins</p>
+      <div>
+        <ul>
+          <li>Comment</li>
+          <li>Remove</li>
+          <li>Edit</li>
+        </ul>
+      </div>
+    </div>
+    <div className="book-end-ration">
```
Comment (dasileker) [inline src/components/Book.js]: - [ ] The books should be called props from your addbook.

File: src/components/Books.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Book from './Book';
+import AddBook from './AddBook';
+
+const Books = () => (
+  <div>
+    <Book />
+    <Book />
+    <Book />
+    <AddBook />
```
Comment (dasileker) [inline src/components/Books.js]: - [ ] since you are calling books here please make them as prop-types.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @AtillaTahak,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ArthurGC — APPROVED** [review]
# Approved project!  💯 ✅ 🥳 🎈 🎉
Hi  @AtillaTahak 

Amazing job implementing suggestions from the previous reviewer! 💯 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**